runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/testing/TestDataflowRunnerTest.java

```diff
@@ -132,8 +140,8 @@ public void testRunBatchJobThatSucceeds() throws Exception {
     when(mockRunner.run(any(Pipeline.class))).thenReturn(mockJob);
 
     TestDataflowRunner runner = (TestDataflowRunner) p.getRunner();
-    when(request.execute()).thenReturn(
-        generateMockMetricResponse(true /* success */, true /* tentative */));
+    when(request.execute()).thenReturn(generateMockMetricResponse(true /* success */,
```

Comment:
You have several places where you use the null 'otherMetrics' argument instead of the two argument version of generateMockMetricResponse, why?